index.html

```diff
@@ -75,133 +77,137 @@ <h1 class="hero-heading">Build better processes</h1>
         <!-- <div class="container-common"> -->
-            <div class="trending-technologies-main-container">
-                <div class="trending-technologies-text-content">
-                    <h6 class="title-pink">TRENDING TECHNOLOGIES</h6>
-                    <h3 class="title-sub">Popular topics
-                        to learn now</h3>
-                    <p>
-                        Not sure what technologies to focus on? The Technology Index ranks 850+ technologies by their
-                        relative
-                        popularity. See what’s topping the charts and get recommended courses to skill up.
-                    </p>
-                    <div class="arrow-path-container">
-                        <div class="arrow-path-text">
-                            <p>View the index</p>
-                        </div>
-                        <div class="right-arrow">
-                            <span><img src="./images/chevron-right.svg" alt=""></span>
-                        </div>
+        <div class="trending-technologies-main-container">
+            <div class="trending-technologies-text-content">
+                <h6 class="title-pink">TRENDING TECHNOLOGIES</h6>
+                <h3 class="title-sub title-small">Popular topics
+                    to learn now</h3>
+                <p>
+                    Not sure what technologies to focus on? The Technology Index ranks 850+ technologies by their
+                    relative
+                    popularity. See what’s topping the charts and get recommended courses to skill up.
+                </p>
+                <div class="arrow-path-container">
+                    <div class="arrow-path-text">
+                        <p>View the index</p>
+                    </div>
+                    <div class="right-arrow">
+                        <span><img src="./images/chevron-right.svg" alt=""></span>
                     </div>
                 </div>
+            </div>
 
-                <div class="program-study-elements-container">
-                    <div class="program-study-elements">
-                        <div class="program-study-path-wrapper">
-                            <div class="program-study-path-content">
-                                <div class="content-path">
-                                    <span>PATH</span>
-                                </div>
-                                <div class="content-course">
-                                    <span>JavaScript</span>
-                                </div>
-                                <div class="content-courses-number-container">
-                                    <span><span class="content-courses-number">11 </span>courses</span>
+            <div class="program-study-elements-container">
+                <div class="program-study-elements">
+                    <div class="program-study-path-wrapper">
+                        <div class="program-study-path-content">
```

Comment:
These names are kinda confusing (you kinda switch between, program-study-elements and program-study-path-content). Try to have like... "program-study-elements-container", "program-study-elements", "program-study-element" (non plural), "program-study-element-content"... This would make more sense because of hierarchy.

Reply:
done